R/processing.R

```diff
@@ -831,13 +856,24 @@ run_qgis <- function(alg = NULL, params = NULL, check_params = TRUE,
                       overwrite_layer = TRUE)
       # return the result
       file.path(tmp_dir, paste0(names(params)[[i]], ".shp"))
+    } else if (any(tmp %in% c("sf", "sfc", "sfg"))) {
+      # st_write cannot currently replace layers, so file.remove() them
+      file.remove(list.files(path = tmp_dir, 
+                             pattern = names(params)[[i]],
+                             full.names = TRUE))
+      sf::st_write(params[[i]], 
```

Comment:
remove package tag `::` -> not used in R packages.  
Specify single functions in the documentation using `@importFrom sf st_write`